examples/deadline/All-In-AWS-Infrastructure-SEP/ts/lib/sep-stack.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  SecurityGroup,
+  Vpc,
+} from '@aws-cdk/aws-ec2';
+import {
+  Construct,
+  Duration,
+  Stack,
+  StackProps
+} from '@aws-cdk/core';
+import {
+  ManagedPolicy,
+  Role,
+  ServicePrincipal
+} from '@aws-cdk/aws-iam';
+import {
+  RenderQueue,
+  Repository,
+  Stage,
+  ThinkboxDockerRecipes,
+} from 'aws-rfdk/deadline';
+
+/**
+ * Properties for {@link SEPStack}.
+ */
+export interface SEPStackProps extends StackProps {
+
+  /**
+   * The path to the directory where the staged Deadline Docker recipes are.
+   */
+  readonly dockerRecipesStagePath: string;
+}
+
+export class SEPStack extends Stack {
+
+  /**
+   * Initializes a new instance of {@link NetworkTier}.
+   * @param scope The scope of this construct.
+   * @param id The ID of this construct.
+   * @param props The stack properties.
+   */
+  constructor(scope: Construct, id: string, props: SEPStackProps) {
+    super(scope, id, props);
+    
+    //
+    // Network Tier
```

Comment:
We call out the network tier here, but none of the other tiers, which could make it confusing as to what this is actually referring to.